internal/amqpproxy/amqp_proxy_test.go

```diff
@@ -77,16 +77,15 @@ func mustCreateAMQPProxy(t *testing.T) testAMQPProxy {
 		os.RemoveAll(dir)
 	})
 
-	jsonlFile := filepath.Join(dir, "amqpproxy-traffic")
-
 	env := testhelpers.InitLiveTests("../..")
 
 	amqpProxy, err := amqpproxy.NewAMQPProxy(
 		"localhost:5671",
 		env.ServiceBusEndpoint,
 		&amqpproxy.AMQPProxyOptions{
-			BaseJSONName: jsonlFile,
-			CertDir:      dir,
+			LogFolder:  dir,
+			EnableJSON: true,
```

Comment:
does this need to be passed in? isn't it the default?

Reply:
From the APIs perspective, it's not the default ,although it is the default in the CLI itself.